pgtype/convert.go

```diff
@@ -149,7 +149,7 @@ func underlyingTimeType(val interface{}) (interface{}, bool) {
 	switch refVal.Kind() {
 	case reflect.Ptr:
 		if refVal.IsNil() {
-			return time.Time{}, false
+			return nil, false
```

Comment:
This change seems unrelated. Is it intentional?

Reply:
Hi, sorry for the late reply.

Yes, it is intentional, it's in a separate commit. (I could have made a separate PR but it's so minor...) The first return value is ignored when the second is false - so there's no point heap-allocating a zero time.Time that will be thrown away by the caller. It's possibly just a mistake/holdover from when the function's return signature was `(time.Time,bool)`?